src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random number = new Random();
+        ColorSupplier color = new ColorSupplier();
```

Comment:
lets make Random and ColorSupplier class level variables

Reply:
Can you explain this, I can't catch a point)